### DISCUSSION TITLE
Intégrer les prévoyances complémentaire

### DISCUSSION THREAD

**Discussion by johangirod** [betagouv/mon-entreprise#2606] [Idées]
Les contributions patronales de retraite supplémentaire et de prévoyance complémentaire sont exclues de l'assiette des cotisations sous [certaines conditions](https://www.urssaf.fr/portail/home/employeur/calculer-les-cotisations/les-elements-a-prendre-en-compte/la-prevoyance-complementaire.html). Il faudrait, comme pour la mutuelle, demander le montant et la part employeur et si elles obéissent à ces conditions.

- si oui : la part employeur est éxoneré de cotiz sous le même modèle que la mutuelle
- si non : la part employeur est considéré comme avantage en nature

**Comment by mquandalle** [comment]
Demande utilisateur d'intégrer la retraite supplémentaire pour un dirigeant de SASU. Pour info il y a un début d'implémentation dans la base de règle que j'avais désactivé à cause d'une boucle sur le calcul d'un plafond.
